packages/playground/src/portal/transfer_view.vue

```diff
@@ -0,0 +1,241 @@
+                #="{ props }"
+              >
+                <input-tooltip tooltip="Enter Twin ID of Receipient Account">
+                  <v-text-field label="Recipient TwinID:" v-bind="props" v-model="receipientTwinId"></v-text-field>
+                </input-tooltip>
+              </input-validator>
+
+              <input-validator
+                v-if="!loadingBalance"
+                :value="transferAmount"
+                :rules="[
+                  validators.required('Transfer amount is required '),
+                  validators.isNumeric('Amount should be a number.'),
+                  validators.min('Amount must be greater than 0', 0.00000000001),
+                  validators.max('Insufficient funds', freeBalance),
+                ]"
+                #="{ props }"
+              >
+                <input-tooltip tooltip="0.01 transaction fee will be deducted">
+                  <v-text-field label="Transfer Amount:" v-bind="props" v-model.number="transferAmount"></v-text-field>
+                </input-tooltip>
+              </input-validator>
+              <strong v-else>Loading...</strong>
+            </form-validator>
+            <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn @click="clearInput" class="primary white--text">Clear</v-btn>
+              <v-btn
+                class="primary white--text"
+                :loading="loadingTwinIDTransfer"
+                :disabled="!isValidTwinIDTransfer"
+                @click="submitFormTwinID"
+                color="secondary"
+                variant="tonal"
+                >Send</v-btn
+              >
+            </v-card-actions>
+          </v-card>
+        </v-window-item>
+        <!-- Address Transfer -->
+
+        <v-window-item :value="1">
+          <v-card class="pa-5 my-5" flat>
+            <form-validator v-model="isValidAddressTransfer">
+              <input-validator
+                :value="receipientAddress"
+                :rules="[
+                  validators.required('Receipient address is required '),
+                  validators.isAlphanumeric('Invalid Address'),
+                  isSameAddress,
+                ]"
+                :async-rules="[isValidAddress]"
+                #="{ props }"
+              >
+                <input-tooltip tooltip="Enter Address of Receipient Account">
+                  <v-text-field label="Recipient Address:" v-model="receipientAddress" v-bind="props"> </v-text-field>
+                </input-tooltip>
+              </input-validator>
+              <input-validator
+                v-if="!loadingBalance"
+                :value="transferAmount"
+                :rules="[
+                  validators.required('Transfer amount is required '),
+                  validators.isNumeric('Amount should be a number.'),
+                  validators.min('Amount must be greater than 0', 0.00000000001),
+                  validators.max('Insuffient funds', freeBalance),
```

Comment:
```suggestion
                  validators.max('Insufficient funds', freeBalance),
```

Reply:
I really don't get why it shows up like this for you still been fixed in an earlier commit 
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/56790126/407ed0d7-721d-4ddb-bf97-effee7d98aa5)